modules/4337/src/utils/userOp.ts

```diff
@@ -80,19 +80,19 @@ export const buildSafeUserOp = (template: OptionalExceptFor<SafeUserOperation, '
   return {
     safe: template.safe,
     nonce: template.nonce,
-    initCode: template.initCode || '0x',
-    callData: template.callData || '0x',
-    callGasLimit: template.callGasLimit || 2000000,
-    verificationGasLimit: template.verificationGasLimit || 500000,
-    preVerificationGas: template.preVerificationGas || 60000,
+    initCode: template.initCode ?? '0x',
```

Comment:
If you pass a falsy value here, let's say (0) the `||` operator would default to the default value, the `??` operator only checks for null and undefined without coercing the value to a boolean